src/service/action-impl.js

```diff
@@ -229,8 +228,7 @@ export class ActionService {
           if (!event.defaultPrevented &&
               element.getAttribute('role') == 'button') {
             event.preventDefault();
-            // TODO(choumx, #9699): HIGH.
-            this.trigger(element, name, event, ActionTrust.MEDIUM);
+            this.trigger(element, name, event, ActionTrust.HIGH);
```

Comment:
@jridgewell FYI I think this is the correct trust level for clicking a button via keyboard.